packages/fast-usdc-contract/src/utils/store.ts

```diff
@@ -113,3 +113,51 @@ export const asMultiset = <K extends Key>(mapStore: MapStore<K, number>) =>
       mapStore.clear();
     },
   });
+
+/**
+ * Append `item` to the array held at `key`, keeping it **sorted** according
+ * to `compare`.  Uses a simple linear scan – clearer than a binary search and
+ * plenty fast for small lists.
+ *
+ * Stable‑after: duplicates are inserted *after* existing equal elements.
+ *
+ * @template K - Type of keys in the map store
+ * @template V - Type of values in the arrays stored in the map
+ */
+export const appendToSortedStoredArray = <K, V>(
+  mapStore: MapStore<K, V[]>,
+  key: K,
+  item: V,
+  compare: (a: V, b: V) => number = defaultCompareDesc as any,
```

Comment:
I think the `any` is hiding that `V` requires this same constraint, `bigint | number | string`, as `defaultCompareDesc` has or the default isn't valid. 

I see the one use of this is to compare transactions, which takes a comparator. So please remove this default one to make it sound.